### PR TITLE
Enable profile "cloud" by default

### LINKED ISSUE #3201: Github actions fail after introduction of configurations for Cloud Bundle
## Required Information

* [ ] AEM Version, including Service Packs, Cumulative Fix Packs, etc: _______
* [X] ACS AEM Commons Version: 6.3.0
* [X] Reproducible on Latest? yes

### Expected Behavior

Github actions do not fail

### Actual Behavior

Github actions fail.

### Links

See Github action logs: https://github.com/Adobe-Consulting-Services/acs-aem-commons/actions/runs/6471397505

### PR DESCRIPTION
**Pull request by kwin**
Document disabling cloud profile
Fix GH actions build

This closes #3201

### CODE REVIEW

**Comment by kwin** [comment]
@royteeuwen Can you have a look at the PR?

File: pom.xml [hunk 1]
```diff
@@ -913,7 +916,6 @@ Service-Component: OSGI-INF/*.xml
 
     <modules>
         <module>bundle</module>
-        <module>bundle-cloud</module>
```
Comment (royteeuwen) [inline pom.xml:916]: why did you remove this module from the reactor?

**Review by royteeuwen — APPROVED** [review]
LGTM!

**Comment by kwin** [comment]
For some weird reason the build order is wrong:

```
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for ACS AEM Commons - Reactor Project 6.3.1-SNAPSHOT:
[INFO] 
[INFO] ACS AEM Commons - Reactor Project .................. SUCCESS [  0.220 s]
[INFO] ACS AEM Commons Bundle ............................. SUCCESS [ 14.504 s]
[INFO] ACS AEM Commons Oakpal Checks ...................... SUCCESS [  1.347 s]
[INFO] ACS AEM Commons UI.Apps Package .................... SUCCESS [  4.820 s]
[INFO] ACS AEM Commons UI.Content Package ................. SUCCESS [  0.220 s]
[INFO] ACS AEM Commons Package ............................ SUCCESS [  0.775 s]
[INFO] ACS AEM Commons Bundle - Core Cloud Fragment ....... SUCCESS [  0.242 s]
[INFO] ------------------------------------------------------------------------
```

That is despite the fact hat ACS AEM Commons Package should depend on the Core Cloud Fragment (in the profile `cloud`).

**Comment by kwin** [comment]
I created https://issues.apache.org/jira/browse/MNG-7926 for the ordering issue.

**Comment by kwin** [comment]
Actually it was just a misunderstanding from my side, the `activeByDefault` does no activate all profiles with the same id. This activation needs to be part of each (same named) profile, as profiles are not inherited. After activating all profiles `cloud` by default it works as expected.

**Comment by davidjgonzalez** [comment]
@kwin should this PR's build work? Or will this only work once we merge to main?
